recipe/build.sh

```diff
@@ -207,7 +207,7 @@ build --define=PROTOBUF_INCLUDE_PATH=${PREFIX}/include
 # TODO: re-enable once we upgrade from gcc 11.8 and get support for flag -mavx512fp16.
 # See: https://github.com/google/XNNPACK/blob/master/BUILD.bazel
 build --define=xnn_enable_avx512fp16=false
-build --config=noaws
+build --define=xnn_enable_avxvnniint8=false
```

Comment:
This is required because AVX-VNNI-INT8 was added in GCC13 https://gcc.gnu.org/gcc-13/changes.html